backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/emx2/Emx2Members.java

```diff
@@ -20,6 +18,8 @@ private Emx2Members() {
   }
 
   public static void outputRoles(TableStore store, Schema schema) {
+    checkRoles(schema);
```

Comment:
this is the right level to shield the export, however, it should simply not output, not throw exceptions.

You should here asked  if(schema.getRoleForActiveUser() ....) and then check the if suitable role otherwise not output.